.changeset/eighty-rings-pull.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/backend-deployer': patch
+'@aws-amplify/platform-core': minor
```

Comment:
looks like there are a few other packages that need to be included in this changeset